iavl/visualize.py

```diff
@@ -28,15 +28,26 @@ def get_node(hash: bytes) -> Node:
         n, _ = decode_node(db.get(prefix + node_key(hash)), hash)
         return n
 
-    def vis_node(hash: bytes, n: Node):
-        style = "solid" if n.version == version else "filled"
-        g.node(HexBytes(hash).hex(), label=label(node), style=style)
-
+    def vis_node(hash: bytes, n: Node, new=False):
+        # assign style when new=True
+        fillcolor = None
+        style: str
+        if new:
+            style = "filled"
+            fillcolor = "#bed8a0"
+        elif n.version == version:
+            style = "solid"
+        else:
+            style = "filled"
```

Comment:
What does these three status means?

Reply:
1. new
2. existing with bump up version
3. old

Reply:
I see `node.key not in last_keys`, but what's the point to compare this, or what insights does this give us?